Tests/IntentionalNilEncodingTests.swift

```diff
@@ -0,0 +1,247 @@
+//
+//  IntentionalNilEncodingTests.swift
+//
+//  Copyright (c) 2014-2018 Alamofire Software Foundation (http://alamofire.org/)
```

Comment:
All new files should use the current year for copyright.

Reply:
Really, though, these tests should be consolidated into the existing encoding tests.